diff --git a/ecell4/bd/tests/CMakeLists.txt b/ecell4/bd/tests/CMakeLists.txt
index 2d2c81c7..ada18f90 100644
--- a/ecell4/bd/tests/CMakeLists.txt
+++ b/ecell4/bd/tests/CMakeLists.txt
@@ -6,6 +6,10 @@ if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_definitions(-DBOOST_TEST_DYN_LINK)
     add_definitions(-DUNITTEST_FRAMEWORK_LIBRARY_EXIST)
     set(test_library_dependencies ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
+else()
+    if(UNIX AND NOT APPLE)
+        set(test_library_dependencies rt)
+    endif()
 endif()
 
 foreach(TEST_NAME ${TEST_NAMES})
diff --git a/ecell4/core/tests/CMakeLists.txt b/ecell4/core/tests/CMakeLists.txt
index 30d0de8b..fee35cb6 100644
--- a/ecell4/core/tests/CMakeLists.txt
+++ b/ecell4/core/tests/CMakeLists.txt
@@ -13,6 +13,10 @@ if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_definitions(-DBOOST_TEST_DYN_LINK)
     add_definitions(-DUNITTEST_FRAMEWORK_LIBRARY_EXIST)
     set(test_library_dependencies ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
+else()
+    if(UNIX AND NOT APPLE)
+        set(test_library_dependencies rt)
+    endif()
 endif()
 
 foreach(TEST_NAME ${TEST_NAMES})
diff --git a/ecell4/gillespie/tests/CMakeLists.txt b/ecell4/gillespie/tests/CMakeLists.txt
index f6f0fc4f..a5bcbe93 100644
--- a/ecell4/gillespie/tests/CMakeLists.txt
+++ b/ecell4/gillespie/tests/CMakeLists.txt
@@ -6,6 +6,10 @@ if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_definitions(-DBOOST_TEST_DYN_LINK)
     add_definitions(-DUNITTEST_FRAMEWORK_LIBRARY_EXIST)
     set(test_library_dependencies ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
+else()
+    if(UNIX AND NOT APPLE)
+        set(test_library_dependencies rt)
+    endif()
 endif()
 
 foreach(TEST_NAME ${TEST_NAMES})
diff --git a/ecell4/meso/tests/CMakeLists.txt b/ecell4/meso/tests/CMakeLists.txt
index 596293b2..132cc189 100644
--- a/ecell4/meso/tests/CMakeLists.txt
+++ b/ecell4/meso/tests/CMakeLists.txt
@@ -6,6 +6,10 @@ if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_definitions(-DBOOST_TEST_DYN_LINK)
     add_definitions(-DUNITTEST_FRAMEWORK_LIBRARY_EXIST)
     set(test_library_dependencies ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
+else()
+    if(UNIX AND NOT APPLE)
+        set(test_library_dependencies rt)
+    endif()
 endif()
 
 foreach(TEST_NAME ${TEST_NAMES})
diff --git a/ecell4/ode/tests/CMakeLists.txt b/ecell4/ode/tests/CMakeLists.txt
index f9cd2c21..c8c13c40 100644
--- a/ecell4/ode/tests/CMakeLists.txt
+++ b/ecell4/ode/tests/CMakeLists.txt
@@ -6,6 +6,10 @@ if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_definitions(-DBOOST_TEST_DYN_LINK)
     add_definitions(-DUNITTEST_FRAMEWORK_LIBRARY_EXIST)
     set(test_library_dependencies ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
+else()
+    if(UNIX AND NOT APPLE)
+        set(test_library_dependencies rt)
+    endif()
 endif()
 
 foreach(TEST_NAME ${TEST_NAMES})
diff --git a/ecell4/spatiocyte/tests/CMakeLists.txt b/ecell4/spatiocyte/tests/CMakeLists.txt
index 0d717470..4cf80a3a 100644
--- a/ecell4/spatiocyte/tests/CMakeLists.txt
+++ b/ecell4/spatiocyte/tests/CMakeLists.txt
@@ -8,6 +8,10 @@ if (Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_definitions(-DBOOST_TEST_DYN_LINK)
     add_definitions(-DUNITTEST_FRAMEWORK_LIBRARY_EXIST)
     set(test_library_dependencies ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
+else()
+    if(UNIX AND NOT APPLE)
+        set(test_library_dependencies rt)
+    endif()
 endif()
 
 foreach(TEST_NAME ${TEST_NAMES})
